File: app/src/main/java/com/example/appmovil/Publicacion.java
package com.example.appmovil;

import android.media.Image;

import java.io.Serializable;
import java.sql.Date;

public class Publicacion implements Serializable {
    private int id_publicacion;
    private int id_usuario;
    private Date fecha_publicacion;

    private Image imagen;

    private String descripcion;
    private int likes;
    private int comentarios;


}

File: App_escritorio/Klyer/Publicacion.cs
using System;

namespace Klyer
{
    /// <summary>
    /// Modelo de datos para una publicación del muro.
    /// Las propiedades usan nombres en minúsculas para coincidir con el JSON de la API.
    /// </summary>
    public class Publicacion
    {
        public int id_publicacion { get; set; }
        public int id_usuario { get; set; }
        public string fecha_publicacion { get; set; }
        public string imagen { get; set; }
        public string descripcion { get; set; }
        public int likes { get; set; }
        public int comentarios { get; set; }
        public string nickname_usuario { get; set; }
        public string foto_usuario { get; set; }
    }
}
